src/Services/LoggingService.php

```diff
@@ -116,17 +120,35 @@ private function getPlentyMarketLoggerInstance(string $method, string $reference
   }
 
   /**
-   * Maps data from the he associative array that Wayfair provides to the PlentyMarkets logging API's inputs
+   * Maps data from the associative array that Wayfair provides to the PlentyMarkets logging API's inputs
    *
    * @param $loggingInfo
    *
    * @return array
    */
   public function extractVars($loggingInfo): array {
-    $additionalInfo = $loggingInfo['additionalInfo'] ?? [];
+    $truncatedMsg =[];
+    $externalLogs = pluginApp(ExternalLogs::class);
+
+    // $additionalInfo = $loggingInfo['additionalInfo'] ?? [];
     $method = $loggingInfo['method'] ?? null;
     $referenceType = $loggingInfo['referenceType'] ?? null;
     $referenceValue = (int) $loggingInfo['referenceValue'] ?? null;
```

Comment:
what is this for?
The reference value will not always be an integer. It is whatever the developer decides to supply as a reference value.

Reply:
Right. I didn't mean to make this a change request. It was a question about the code around your changes.
I looked up the API and it looks like referenceValue is supposed to be an integer. Interestingly it's also deprecated.

Reply:
I am not sure why this was cast will look into it